main.css

```diff
@@ -349,183 +220,95 @@
     cursor: pointer;
     z-index: 2;
 }
-
 /* LET'S DO SOFTWARE */
+.first-page { height: 96.5vh; }
 .lets-do-image {
-    background-image: url('https://cdn.rawgit.com/dynobjx/dynobjx.github.io/master/02_Image.png');
+    background-image: url('rsz_02_image.png');
```

Comment:
Use the raw git file directory (cdn-based) of this image. Again, using the commit hash URL.